### DISCUSSION TITLE
CNV outliers between two conditions

### DISCUSSION THREAD

**Discussion by JorgeCortes-M** [piyalkarum/rCNV#5] [Q&A] [answered]
Hi Piyal,

I hope you are doing well! I am being using the package to analyze data related to fish exposed to pollution. After the CNV detection I wanted to find those CNV that differentiate most the populations between polluted and non polluted conditions. So I applied a permutation t-test over the normalized read depth of the CNV detected between polluted and non polluted populations. But now I was thinking also to run a DAPC to find this kind of CNV outliers. What do you think about it, it is a feasible approach taking in consideration the characteistics of the data?

Thank you again for your help!

**Comment by piyalkarum** [comment, marked as answer]
Hi Jorge,

Thanks for the continued use of the package and feedback. 
I like your ideas of permutation t-test and DAPC on the depth of filtered CNVs. Especially, with DAPC, you will most likely get a good definition of the two groups if you have a clear depth variation between the polluted and non-polluted groups. You may have seen this variation with the t-test. However, I would be careful when interpreting the results though. My concern is that if you have a high copy number variation (more than two copies) in the polluted group and there are individuals/populations in the non-polluted group with more than one copy and less than the number of copies in the polluted group, the depth variation might not be enough for a DAPC to discriminate the non-polluted individuals with more than one copy. Your t-test however should give you a clue about the distribution of the depth in the two groups.
I hope this helps.

Piyal

**Reply by JorgeCortes-M** [reply]
Hi Piyal,

Thank you for the advice, I will try with both methods to test how they perform!
